test/dummy/app/controllers/cache_controller.rb

```diff
@@ -1,5 +1,5 @@
 class CacheController < ApplicationController
-  before_filter :instrument_caching
+  Librato::Rails::VersionSpecifier.supported(max: '4.1') { before_filter :instrument_caching }
```

Comment:
Ah, makes sense - maybe add a comment to this effect for future selves? Would be nice to make it clear that it isn't needed after 4.1 because it is always on after that, we aren't actually removing something.